lib/beaker-aws/version.rb

```diff
@@ -1,3 +1,3 @@
 module BeakerAws
-  VERSION = '0.9.0'
+  VERSION = '0.9.1'
```

Comment:
please don't update the version, we identify the next release version during the release process

Reply:
cool. reverted back.